src/main/java/io/reactivex/disposables/CompositeDisposable.java

```diff
@@ -86,7 +86,7 @@ public boolean isDisposed() {
 
     @Override
     public boolean add(Disposable d) {
-        ObjectHelper.requireNonNull(d, "d is null");
+        ObjectHelper.requireNonNull(d, "Disposable is null");
```

Comment:
Alright but don't you think that the error message `Disposable is null` is better than `d is null` regardless of the parameter name?